### PR TITLE
merge lexy-server-error-config to main

### PR DESCRIPTION
**Pull request by alexamaingard**

### CODE REVIEW

File: src/server/config.js [hunk 1]
```diff
@@ -2,13 +2,20 @@ const EXTERNAL_API = 'https://pokeapi.co/api/v2/pokemon/';
 
 const SECRET = process.env.SECRET;
 
-const SERVER_ERROR_MESSAGE = {
```
Comment (bravint) [inline src/server/config.js:5]: happy with the changes as suggested in PR #23

File: src/server/controllers/init.js [hunk 2]
```diff
@@ -33,13 +34,13 @@ const initPokemonDatabase = async (req, res) => {
         const response = await axios(`${EXTERNAL_API}${pokemonId}`);
         const fetchedPokemon = response.data;
 
-        const filteredPokemon = await filterPokemonData(fetchedPokemon,getPokemonById);
+        const filteredPokemon = await filterPokemonData(fetchedPokemon, pokemonId);
```
Comment (polarbear23) [inline src/server/controllers/init.js:37]: nice fix

Comment (bravint) [reply src/server/controllers/init.js:37]: + 1

**Review by bravint — APPROVED** [review]
Nice work, changes applied as agreed

**Review by polarbear23 — APPROVED** [review]

**Review by hannah68 — APPROVED** [review]
nice work

**Pull request merged**